R/dsieur_ch5n6_script.R

```diff
@@ -58,3 +58,5 @@ ma_data_init %>%
   count() %>% 
   filter(n > 10) %>% 
   arrange()
+
+yxwexnwe
```

Comment:
Hi Mark - is this last bit intentional

Reply:
Hi all! Mark, is this to show how to debug?

Reply:
```suggestion

```